src/components/modals/details-qr.tsx

```diff
@@ -28,80 +29,82 @@ export interface DetailsQrModalProps {
   address: string;
 }
 
-export const DetailsQrModal = ({address, onClose}: Modals['cardDetailsQr']) => {
-  const svg = useRef();
-  const wallet = useWallet(address) as Wallet;
-  const {width} = useWindowDimensions();
+export const DetailsQrModal = observer(
+  ({address, onClose}: Modals['cardDetailsQr']) => {
+    const svg = useRef();
+    const wallet = useWallet(address) as Wallet;
```

Comment:
`useWallet` already has `Wallet` type. Why you are using `as Wallet` here?

Reply:
useWallet can return Wallet or null. Type assertion here and in other places were used in previous versions of our app and I just store this behaviour. But anyway in completed PR I'll mark it with TODOs to check it as technical debt. To many changes for 1 PR